sormas-api/src/main/java/de/symeda/sormas/api/caze/ICase.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2024 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.symeda.sormas.api.caze;
+
+import de.symeda.sormas.api.uuid.HasUuid;
+
+public interface ICase extends HasUuid {
```

Comment:
Not a fan of this name because we don't use the (in my eyes questionable) convention of naming interfaces with a prefix "I". I'd suggest "IsCase" because it uses a similar pattern as "HasUuid" does.